mlir/test/Dialect/ArmSME/vector-legalization.mlir

```diff
@@ -646,3 +646,11 @@ func.func @negative_transpose_store_scalable_via_za__bad_source_shape(%vec: vect
   vector.transfer_write %tr, %dest[%i, %j] {in_bounds = [true, true]} : vector<[7]x2xf32>,  memref<?x?xf32>
   return
 }
+
+// -----
+
+// From: https://github.com/llvm/llvm-project/issues/118449 (check we don't crash).
```

Comment:
Every test is effectively checking that the compiler doesn't crash for the edge case being exercise ;-)

Please update this comment with info that clarifies what makes this case unique.